demisto_sdk/commands/lint/helpers.py

```diff
@@ -281,18 +281,26 @@ def add_tmp_lint_files(content_repo: git.Repo, pack_path: Path, lint_files: List
 
 
 @lru_cache(maxsize=100)
-def get_python_version_from_image(image: str) -> float:
+def get_python_version_from_image(image: str, timeout: int = 60) -> float:
     """ Get python version from docker image
 
     Args:
         image(str): Docker image id or name
+        timeout(int): Docker client request timeout
 
     Returns:
         float: Python version X.Y (3.7, 3.6, ..)
     """
+
+    excluded_images = {'demisto/powershell', 'demisto/powershell-ubuntu', 'demisto/pwsh-exchange',
+                       'demisto/pwsh-infocyte', 'demisto/powershell-teams'}
+    is_pwsh_image = any([image_name in image for image_name in excluded_images])
+    if is_pwsh_image:
+        return 3.8
```

Comment:
why 3.8?

Reply:
seems to be the default, really does not matter when running on pwsh